src/entities/CustomPassword.java

```diff
@@ -0,0 +1,16 @@
+package src.entities;
+
+public class CustomPassword extends Password {
```

Comment:
Is checking the strength part of the password object? It seems like this behaviour belongs in StrengthChecker, maybe?